src/components/InputField/InputField.tsx

```diff
@@ -5,24 +5,44 @@ type InputFieldProps = {
   label: string;
   fieldType: "POSTCODE" | "AMOUNT" | "PERCENTAGE" | "TEXT";
   required: boolean;
+  id: string;
+  data: (value: string, id?: string) => void 
+  className: string;
```

Comment:
If you're going to pass in a className, it'd rather be a list of strings and then consumed in the React component via join, joined into a string with a space in between them.